### PR TITLE
New search filling m_localMwmCounter and m_localMwmSize

### PR DESCRIPTION
**Pull request by bykoianko**
Еще доработка по https://jira.mail.ru/browse/MAPSME-76
@igrechuhin PTAL

### CODE REVIEW

File: storage/storage.cpp [hunk 1]
```diff
@@ -1307,6 +1308,20 @@ void Storage::GetNodeAttrs(TCountryId const & countryId, NodeAttrs & nodeAttrs)
   nodeAttrs.m_downloadingProgress =
       CalculateProgress(downloadingMwm, descendants, downloadingMwmProgress, setQueue);
 
+  // Local mwm information.
+  node->ForEachInSubtree([this, &nodeAttrs](TCountriesContainer const & d)
+  {
+    auto const localeMapIt = m_localFiles.find(d.Value().Name());
+    if (localeMapIt != m_localFiles.end())
+    {
```
Comment (ygorshenin) [inline storage/storage.cpp]: It's better and shorter to do early exit here.

File: storage/storage.cpp [hunk 2]
```diff
@@ -1307,6 +1308,20 @@ void Storage::GetNodeAttrs(TCountryId const & countryId, NodeAttrs & nodeAttrs)
...
+      nodeAttrs.m_localMwmCounter += 1;
+      list<TLocalFilePtr> const & localMwmFiles = localeMapIt->second;
+      ASSERT_EQUAL(localMwmFiles.size(), 1, ("Storage::GetNodeAttrs is called for two component mwms."));
+      nodeAttrs.m_localMwmSize += (*localMwmFiles.begin())->GetSize(MapOptions::Map);
```
Comment (ygorshenin) [inline storage/storage.cpp]: Please, add an assertion that localMwmFiles is not empty.

Comment (ygorshenin) [inline storage/storage.cpp]: Probably you're interested in a latest mwm local size, right?

Comment (ygorshenin) [inline storage/storage.cpp]: BTW, why not to use GetLatestLocalFile?

File: storage/storage.cpp (hunk 1 above)
Comment (bykoianko) [inline storage/storage.cpp]: DONE

File: storage/storage.cpp [hunk 3]
```diff
@@ -1307,6 +1308,20 @@ void Storage::GetNodeAttrs(TCountryId const & countryId, NodeAttrs & nodeAttrs)
   nodeAttrs.m_downloadingProgress =
       CalculateProgress(downloadingMwm, descendants, downloadingMwmProgress, setQueue);
 
+  // Local mwm information.
+  node->ForEachInSubtree([this, &nodeAttrs](TCountriesContainer const & d)
+  {
+    auto const localeMapIt = m_localFiles.find(d.Value().Name());
```
Comment (kshalnev) [inline storage/storage.cpp]: What about to fix name localeMapIt -> localMapIt. It is not about locale, it is about local files, right?

File: storage/storage.cpp (hunk 2 above)
Comment (bykoianko) [inline storage/storage.cpp]: > BTW, why not to use GetLatestLocalFile?

Yes. GetLatestLocalFile is much better. I'll use it!

File: storage/storage.cpp (hunk 3 above)
Comment (bykoianko) [inline storage/storage.cpp]: This code is rewritten.

**Comment by bykoianko** [comment]
@ygorshenin @kshalnev PTAL

File: storage/storage.cpp (hunk 2 above)
Comment (kshalnev) [inline storage/storage.cpp]: (*localMwmFiles.begin()) = localMwmFiles.front() - меньше символов, понятнее. optionally.

**Comment by ygorshenin** [comment]
LGTM

**Comment by kshalnev** [comment]
LGTM

**Pull request merged**